psutil/_psbsd.py

```diff
@@ -423,6 +436,23 @@ def threads(self):
 
     @wrap_exceptions
     def connections(self, kind='inet'):
+        if NETBSD:
+            families, types = conn_tmap[kind]
+            ret = set()
+            rawlist = cext.proc_connections(self.pid)
+            for item in rawlist:
+                fd, fam, type, laddr, raddr, status = item
+                if fam in families and type in types:
+                    try:
+                        status = TCP_STATUSES[status]
+                    except KeyError:
+                        status = TCP_STATUSES[cext.PSUTIL_CONN_NONE]
+                    fam = sockfam_to_enum(fam)
+                    type = socktype_to_enum(type)
+                    nt = _common.pconn(fd, fam, type, laddr, raddr, status)
+                    ret.add(nt)
+            return list(ret)
+
```

Comment:
This should be moved down, after the `if kind not in conn_tmap` check which raises `ValueError`